app/src/main/java/com/cortado/minionify/WelcomeActivity.java

```diff
@@ -0,0 +1,32 @@
+package com.cortado.minionify;
+
+import android.os.Bundle;
+import android.view.Window;
+import android.view.WindowManager;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+public class WelcomeActivity extends AppCompatActivity {
```

Comment:
`ChoiceActivity` or `RoleChoosingActivity` or smth like that would be more fitting. also this doesnt have to show after the user have chosen once - we should save his choice and send him to the correct activity the next time he opens the app. if you wanna do this in another cr it's also fine

Reply:
I will change the naming. 
About saving the choice, I'll do it.

Reply:
nice